rules/frr.mk

```diff
@@ -0,0 +1,9 @@
+# FRRouting (frr) package
+
+FRR_VERSION = 3.1
+export FRR_VERSION
+
+FRR = frr_$(FRR_VERSION)-dev_amd64.deb
```

Comment:
>-dev [](start = 24, length = 4)

why the package name has dev? usually dev pacakge is for the library development files.